src/test/regress/multi_schedule

```diff
@@ -83,6 +83,7 @@ test: multi_task_assignment_policy multi_cross_shard
 test: multi_utility_statements
 test: multi_dropped_column_aliases foreign_key_restriction_enforcement
 test: multi_binary_master_copy_format
+test: multi_materialized_view
```

Comment:
by adding it to an existing line it will run in parallel with other tests, maybe the same line as multi_view?